### PR TITLE
Add username sanitisation to fix CSV injections

### PR DESCRIPTION
**Pull request by NGnius**
This fixes CSV parsing weirdness caused by a giveaway user with commas in their username. These invalid characters are now removed from the username before it is entered into the giveaway log CSV.

The invalid characters can be customised by adding & removing strings from the list `sanitised_characters` in giveaways.json

### CODE REVIEW

File: cogs/giveaways.py [hunk 1]
```diff
@@ -46,7 +47,10 @@ async def log_activity(self, msg):
         if msg.channel.id == self.giveaway_channel:
             # log user activity to csv if in giveaway channel
             has_role = int(self.has_giveaway_role(msg.author))
-            line = f"{int(time.time())},{msg.author.name},{msg.author.id},{has_role}"
+            sanitised_name = msg.author.name
```
Comment (StripedMonkey) [inline cogs/giveaways.py:50]: Why is this required at all? Why are we logging usernames instead of only ids? If the bot is choosing the members there's no reason to worry about writing any user-provided information at all.

Comment (NGnius) [reply cogs/giveaways.py:50]: The bot is not choosing the winner -- it's just recording who has entered. Technically, you can still ping a user by just knowing their id (`<@this_is_an_id>` will work), but it's not very user-friendly and it makes any last-minute verification a bit more difficult (Discord does not make looking up an ID easy).

Anyway, this is what Orelio asked for.

Comment (SteamDeckMods) [reply cogs/giveaways.py:50]: The bot spits out a CSV of users that entered the giveaway and I manually sanitize that list to remove duplicates, fake accounts, and investigate potential cheating before randomly selecting a user. It's not the most efficient way and people could technically accuse me of "rigging" it, but I'm not too concerned about that and manually checking each entrant is a great way to minimize potential cheating. It's time consuming and tedious but I don't mind it to maintain legitimacy in our giveaways.
